plasmapy/dispersion/analytical/stix_.py

```diff
@@ -206,7 +206,7 @@ def stix(
     elif n_i.size == 1:
         n_i = np.repeat(n_i, len(ions))
 
-    species = ions + [Particle("e-")]
+    species = [*ions, Particle("e-")]
```

Comment:
This fix looks clever!  I hadn't thought about this before, but it's more clear that `ions` is a list here.